### ISSUE TITLE
Qt incompatibility issue after packages upgrade

### ISSUE THREAD

**Issue by greinbold** [nomadbsd/NomadBSD#59]
Hi

I just upgraded all of my packages from a local poudriere repository and many DSB tools no longer work.

I got the same error:

```
Cannot mix incompatible Qt library (version 0x50d02) with this library (version 0x50e02)
Abort trap
```

for:

- dsblogoutmgr
- dsbmixer
- dsbbatmon
- dsbexec

and so on.

I have build all the packages in my poudriere (qt and dsb) in the same time.

The version of the needed Qt library is hard-coded?

Regards

**Comment by greinbold** [comment]
Hello

I reported on FreeBSD bugzilla:

- [x11/dsbautostart: crash at launch: Cannot mix incompatible Qt library with this library](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=245784)
- [sysutils/dsbbatmon: crash at launch: Cannot mix incompatible Qt library with this library](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=245785)
- [sysutils/dsblogoutmgr: crash at launch: Cannot mix incompatible Qt library with this library](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=245786)
- [sysutils/dsbmc: crash at launch: Cannot mix incompatible Qt library with this library](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=245788)
- [audio/dsbmixer: crash at launch: Cannot mix incompatible Qt library with this library](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=245790)

Regards

**Comment by mrclksr** [comment]
All Qt5 applications produce the same error after a `pkg upgrade` (latest). This is not specific to the DSB tools.

**Comment by mrclksr** [comment]
Rebuilding x11-themes/qt5-style-plugins should fix the problem.

**Comment by greinbold** [comment]
I saw the same thing with deskutils/owncloudclient

Rebuilding x11-themes/qt5-style-plugins fix the problem, but is a downgrade:

```
Apr 22 09:39:10 elitebook pkg[5434]: qt5-style-plugins downgraded: 5.0.0.23_7 -> 5.0.0.23
```

I will close related bugs on FreeBSD.

Where you found the origin of the issue? I searched on the FreeBSD bugzilla, Google and so more, but I don't found how I can identify the origin.

Thank you @mrclksr

**Comment by mrclksr** [comment]
> Rebuilding x11-themes/qt5-style-plugins fix the problem, but is a downgrade:

I just did a `cd /usr/ports/x11-themes/qt5-style-plugins && make deinstall && make install clean`

> I will close related bugs on FreeBSD.

Thanks.

> Where you found the origin of the issue? I searched on the FreeBSD bugzilla, Google and so more, but I don't found how I can identify the origin.

Since the problem persisted after rebuilding one of the DSB tools, I checked if any of the needed libraries was not updated. This was not the case, but I realized that x11-themes/qt5-style-plugins I am using was not updated, and so I set `QT_QPA_PLATFORMTHEME=""` to disable it, and the Qt5 applications worked again.
